front-end/package.json

```diff
@@ -7,6 +7,12 @@
   },
   "dependencies": {
     "@material-ui/core": "^1.5.0",
+    "eslint": "^5.6.0",
```

Comment:
Eslint would be better in dev dependencies?

Reply:
true